src/models/encounter/useEncounterFieldSchemas.js

```diff
@@ -94,6 +95,6 @@ export default function useSightingFieldSchemas() {
       }),
       ...customFieldSchemas,
     ];
-  }, [siteSettingsVersion]);
+  }, [intl, data, loading, error, siteSettingsVersion]);
```

Comment:
yeah, i mean theoretically if intl updated itself it would re-execute and redo the translations. i'm not sure what would happen if the user switched languages, maybe this would rerun? my bet is it just never changes though